### PR TITLE
chore(other): update some dependencies to latest

### PR DESCRIPTION
**Pull request by daragh-king-genesys**
update some dependencies to latest

### CODE REVIEW

File: packages/genesys-spark-components/src/components/stable/gux-popover-list/gux-popover-list.tsx [hunk 1]
```diff
@@ -51,12 +51,6 @@ export class GuxPopoverList {
   @Prop()
   position: Placement = 'bottom';
 
-  /**
-   * Indicate if the dismiss button is displayed
-   */
-  @Prop()
-  displayDismissButton: boolean;
```
Comment (daragh-king-genesys) [inline packages/genesys-spark-components/src/components/stable/gux-popover-list/gux-popover-list.tsx:58]: I don't see this as a breaking api change as the component should not have this. Also its removal will not break anything as there are other ways to close the popover-list component.

Comment (katie-bobbe-genesys) [reply packages/genesys-spark-components/src/components/stable/gux-popover-list/gux-popover-list.tsx:58]: This change seems like it should be in a separate PR from a dependency bump PR

Comment (daragh-king-genesys) [reply packages/genesys-spark-components/src/components/stable/gux-popover-list/gux-popover-list.tsx]: Ya fairpoint. Removed from this PR.

**Comment by daragh-king-genesys** [comment]
> I am getting the error,
> 
> ```
> [ ERROR ]  TypeScript: src/test/e2eTestUtils.ts:21:21
>            Property 'violations' does not exist on type 'unknown'.
> 
>      L20:    // eslint-disable-next-line
>      L21:    expect(axeResults.violations).toHaveNoViolations(axeScanDetails);
>      L22:  }
> ```
> 
> when running the build locally. Is anyone else getting the same error ?

I did not get that when I was building or testing the branch but I could see it in vscode when I looked at the file. I added a fix for it.

**Review by katie-bobbe-genesys — APPROVED** [review]

**Pull request merged**